atsign-foundation/at_client_sdk
# fix: expose notification params

**- What I did**
- Expose the following notifications params - priority, strategy, notifier, latestN and notificationExpiry.
- Added setUp method in "atclient_notify_test.dart" to set currentAtSign before each test.
**- How I did it**

**- How to verify it**
- Added unit tests to verify priority, strategy, notifier, latestN and notificationExpiry can be set via the NotificationParams.
- Added functional test and end-to-end test to verify notification expiry.

**- Description for the changelog**
* Expose new attributes in NotificationParams.forUpdate method
